platform-modules/batch-models/src/main/scala/org/ekstep/analytics/util/AssessmentReportUtil.scala

```diff
@@ -46,44 +47,55 @@ object AssessmentReportUtil {
       col("content_name").as("contentName"),
       col("reportUrl").as("reportUrl")
     )
-    try {
-      val indexList = ESUtil.getIndexName(alias)
-      val oldIndex = indexList.mkString("")
-      ESUtil.saveToIndex(assessmentReportDF, index)
-      JobLogger.log("Indexing of assessment report data is success: " + index, None, INFO)
-      if (!oldIndex.equals(index)) ESUtil.rolloverIndex(index, alias)
-    } catch {
-      case ex: Exception => {
-        JobLogger.log(ex.getMessage, None, ERROR)
-        ex.printStackTrace()
-      }
-    }
+    ESUtil.saveToIndex(assessmentReportDF, index)
+  }
+
+  def rollOverIndex(index: String, alias: String): Unit = {
+    val indexList = ESUtil.getIndexName(alias)
+    val oldIndex = indexList.mkString("")
```

Comment:
The index list is still concatenated with empty string. Doesn't this needs to have a comma separator?